suites/pacific/rgw/tier-1-extn_rgw.yaml

```diff
@@ -138,6 +138,22 @@ tests:
         script-name: test_sts_using_boto_session_policy.py
         config-file-name: test_sts_using_boto_restricted_session_policy.yaml
         timeout: 500
+  - test:
```

Comment:
Timeout has been removed from quincy, reef and squide but not from pacific.
if there any changes in pacific suite, please remove timeout from all testcase of that suite.
When not sure on the timeout needed

Reply:
@anrao19 
I removed timeout from only my tests in the PR
there are many such places where timeout is present in all of our RGW suites
we can take it in another PR know
why stressing in this PR

Reply:
done, pls review